src/Tickets/Seating/Admin/Events/Associated_Events.php

```diff
@@ -0,0 +1,282 @@
+					$posts_in_term_qv['post_type'] = $post->post_type;
+				}
+				
+				if ( $taxonomy_object->query_var ) {
+					$posts_in_term_qv[ $taxonomy_object->query_var ] = $t->slug;
+				} else {
+					$posts_in_term_qv['taxonomy'] = $taxonomy;
+					$posts_in_term_qv['term']     = $t->slug;
+				}
+				
+				$label = esc_html( sanitize_term_field( 'name', $t->name, $t->term_id, $taxonomy, 'display' ) );
+				
+				$term_links[] = $this->get_edit_link( $posts_in_term_qv, $label );
+			}
+			
+			echo wp_kses(
+				implode( wp_get_list_item_separator(), $term_links ),
+				[
+					'a' => [
+						'href' => [],
+					],
+				]
+			);
+		}
+	}
+	
+	/**
+	 * Render the attendee count column.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post $item The current WP_Post object.
+	 *
+	 * @return void
+	 */
+	public function column_tickets( $item ): void {
+		$attendee_column = new Tribe__Tickets__Admin__Columns__Tickets( $item->ID );
+		$attendee_column->render_column( 'tickets', $item->ID );
+	}
+	
+	/**
+	 * The sortable columns.
+	 *
+	 * @since TBD
+	 *
+	 * @return array[]
+	 */
+	public function get_sortable_columns() {
+		return [
+			'title' => [ 'title', true ],
+			'date'  => [ 'date', true ],
+		];
+	}
+	
+	/**
+	 * The screen setup.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function screen_setup() {
+		$columns               = $this->get_columns();
+		$hidden                = [];
+		$sortable              = $this->get_sortable_columns();
+		$this->_column_headers = [ $columns, $hidden, $sortable ];
+		
+		add_filter( 'quick_edit_enabled_for_post_type', '__return_false' );
+	}
+	
+	/**
+	 * Prepare the items.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function prepare_items() {
+		$this->screen_setup();
+		
+		$per_page  = 20;
+		$layout_id = tribe_get_request_var( 'layout' );
+		$page      = absint( tribe_get_request_var( 'paged', 0 ) );
+		$orderby   = tribe_get_request_var( 'orderby' );
+		$order     = tribe_get_request_var( 'order' );
+		$search    = tribe_get_request_var( 's' );
+		
+		$arguments = [
+			'status'         => 'any',
+			'paged'          => $page,
+			'posts_per_page' => $per_page,
+		];
+		
+		if ( ! empty( $orderby ) ) {
+			$arguments['orderby'] = $orderby;
+		}
+		
+		if ( ! empty( $order ) ) {
+			$arguments['order'] = $order;
+		}
+		
+		if ( ! empty( $search ) ) {
+			$arguments['s'] = $search;
+		}
+		
+		$ticketable_post_types = Tickets::instance()->post_types();
+		$repository            = new class( $ticketable_post_types ) extends \Tribe__Repository {
+			/**
+			 * @param string[] $post_types The list of post types.
+			 */
+			public function __construct( array $post_types ) {
+				$this->default_args['post_type'] = $post_types;
+				parent::__construct();
+			}
+		};
+		
+		$this->item_count = $repository
+			->where( 'meta_equals', Meta::META_KEY_LAYOUT_ID, $layout_id )
+			->by_args( $arguments )
+			->count();
+		
+		$this->items = $repository
```

Comment:
Fetching **all** posts with one query is a no-go.
There are likely scenarios where there could be a couple layouts on a site and a lot of posts associated with them, in those scenarios, the query could kill the site.
The table should be paginated and, with it, the query that should follow user preferences about the number of posts to show.